.github/workflows/pkgci_sharktank_regression.yml

```diff
@@ -0,0 +1,102 @@
+            target: target_cpu
+            gpu: none
+            runs-on: ubuntu-24.04
+          - name: cpu_llvm_task_regression
+            models-config-file: models_cpu_llvm_task.json
+            backend: cpu
+            runs-on:
+              - self-hosted # must come first
+              - persistent-cache
+              - Linux
+              - X64
+
+          # AMD GPU
+          - name: rocm_hip_model
+            target: target_hip
+            gpu: gfx1100
+            runs-on: nodai-amdgpu-w7900-x86-64
+          - name: amdgpu_rocm_mi250_gfx90a_regression
+            rocm-chip: gfx90a
+            backend: rocm
+            sku: mi250
+            runs-on: nodai-amdgpu-mi250-x86-64
+          - name: amdgpu_rocm_mi300_gfx942_regression
+            rocm-chip: gfx942
+            backend: rocm
+            sku: mi300
+            runs-on: nodai-amdgpu-mi300-x86-64
+          - name: amdgpu_rocm_mi308_gfx942_regression
+            rocm-chip: gfx942
+            backend: rocm
+            sku: mi308
+            runs-on: nodai-amdgpu-mi308-x86-64
+
+    env:
+      VENV_DIR: ${{ github.workspace }}/venv
+      GH_TOKEN: ${{ github.token }}
+      PACKAGE_DOWNLOAD_DIR: ${{ github.workspace }}/.packages
+      TEST_OUTPUT_ARTIFACTS: ${{ github.workspace }}/model_output_artifacts
+    steps:
+      - name: Checking out IREE repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          submodules: false
+
+      - uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38 # v5.4.0
+        with:
+          # Must match the subset of versions built in pkgci_build_packages.
+          python-version: "3.11"
+
+      - uses: actions/download-artifact@cc203385981b70ca67e1cc392babf9cc229d5806 # v4.1.9
+        if: ${{ inputs.artifact_run_id == '' }}
+        with:
+          name: linux_x86_64_release_packages
+          path: ${{ env.PACKAGE_DOWNLOAD_DIR }}
+
+      - name: Setup Python virtual environment
+        run: |
+          ./build_tools/pkgci/setup_venv.py ${VENV_DIR} \
+            --artifact-path=${PACKAGE_DOWNLOAD_DIR} \
+            --fetch-gh-workflow=${{ inputs.artifact_run_id }}
+
+      - name: "Running Sharktank regression composite action"
+        uses: iree-org/iree-test-suites/sharktank_models@775d04553a07a55c64550bb1dd938548adebbe60
+        with:
+          MATRIX_NAME: ${{ matrix.name }}
+          SKU: ${{ matrix.sku }}
+          CHIP: ${{ matrix.rocm-chip }}
+          VENV_DIR: ${{ env.VENV_DIR }}
+          TARGET: ${{ matrix.target }}
+          GPU: ${{ matrix.gpu }}
+          EXTERNAL_TEST_FILE_DIRECTORY: ${{ github.workspace }}/build_tools/pkgci/external_test_suite
+          QUALITY_TEST_DIRECTORY: ${{ github.workspace }}/tests/external/iree-test-suites/sharktank_models/quality_tests
+          BENCHMARK_TEST_DIRECTORY: ${{ github.workspace }}/tests/external/iree-test-suites/sharktank_models/benchmarks
```

Comment:
Per my other comment, I think all of these can be under `iree-test-suites/sharktank_models`